src/usr/local/buildpack/tools/composer.sh

```diff
@@ -24,17 +24,17 @@ if [[ -z "${tool_path}" ]]; then
   base_path=${INSTALL_DIR}/${TOOL_NAME}
   tool_path=${base_path}/${TOOL_VERSION}
 
-  mkdir -p ${tool_path}/bin
+  mkdir -p "${tool_path}"/bin
 
   # OpenShift
-  chmod g+w ${base_path}
+  chmod g+w "${base_path}"
 
   BASE_URL="https://github.com/composer/composer/releases/download"
 
-  curl -sSfLo ${tool_path}/bin/composer ${BASE_URL}/${TOOL_VERSION}/composer.phar
-  chmod +x ${tool_path}/bin/composer
+  curl -sSfLo "${tool_path}"/bin/composer ${BASE_URL}/"${TOOL_VERSION}"/composer.phar
```

Comment:
Shouldn't `${BASE_URL}` be in quotes as well?

Reply:
🤷‍♂️ I think it detects, that it's a simple string without any variables. I don't see any warning in ide

Reply:
```suggestion
  curl -sSfLo "${tool_path}/bin/composer" "${BASE_URL}/${TOOL_VERSION}/composer.phar"
```

Seems better to me